docs/concepts/what-is-a-workflow-execution.md

```diff
@@ -2,57 +2,122 @@
+
+Durability is responsiveness over a period time.
+A Workflow Execution is durable because it executes a Temporal Workflow Definition, also called a Temporal Workflow Function, your application code, effectively once and to completion—whether your code executes for seconds or years.
+Durability is realized through the execution semantics of a Workflow Execution, which are carried out via Temporal's Event loop.
+
+**Reliability**
+
+Reliability is responsiveness in the presence of failure.
+A Workflow Execution is reliable, because it is fully recoverable after a failure.
+To be recoverable is the ability of a process to continue execution after the execution was suspended by a failure.
+Reliability is realized through the execution semantics of a Workflow Execution, which are carried out via Temporal's Event loop.
+
+**Scalability**
+
+Scalability is responsiveness in the presence of load.
+A Workflow Execution is scalable because the Temporal Platform is capable of supporting millions to billions of them concurrently.
+This is realized in part due to the design and nature of a [Temporal Cluster](/docs/concepts/what-is-a-temporal-cluster) and in part due to the design and nature of [Worker Processes](/docs/concepts/what-is-a-worker-process).
+
+### Commands
+
+A Workflow Execution does just two things:
+
+1. Issue [Commands](/docs/concepts/what-is-a-command)
+2. Wait on an Awaitable provided from the generation of Commands.
+
+![Command generation and waiting](/diagrams/workflow-execution-progession-simple.svg)
+
+Commands are issued by the use of Workflow APIs in the [Workflow Definition](/docs/concepts/what-is-a-workflow-definition).
 
-A Workflow Execution has exclusive access to its local state, executes concurrently to all other Workflow Executions, and can communicate with other Workflow Executions using Signals.
+Whenever the Workflow Function is executed, Commands are generated.
+The Worker Process supervises the Command generation and makes sure that it corresponds to the current Event History (See [Deterministic constraints](/docs/concepts/what-is-a-workflow-definition/#deterministic-constraints)).
+The Worker Process batches the Commands and then suspends progress to send the Commands to the Cluster whenever the Workflow Function reaches a place where it can no longer progress without a result from a Awaitable.
 
-A Workflow Execution is either Running or Closed.
-When a Workflow Execution is Running, it is either actively progressing or suspended, awaiting on something.
+Awaitables are provided when using APIs for the following:
 
-![Workflow Execution Running status](/diagrams/workflow-execution-running-status.svg)
+- Spawning a [Child Workflow Execution](/docs/concepts/what-is-a-child-workflow-execution)
+- Sending a [Signal](/docs/concepts/what-is-a-signal)
+- Spawning an [Activity Execution](/docs/concepts/what-is-an-activity-execution)
+- Requesting cancellation of another Workflow Execution
+- Starting a Timer
 
```

Comment:
What about workflow.Await or Future.get? These are not directly linked to any command.